Classes/Task/ReIndexTask.php

```diff
@@ -56,6 +56,13 @@ class ReIndexTask extends AbstractTask
      */
     protected $indexingConfigurationsToReIndex = array();
 
+    /**
+     * Clear Index for selected sites and record types
+     *
+     * @var boolean
+     */
+    protected $clearSearchIndex;
```

Comment:
I'd suggest a default value of `false` just to make sure existing tasks are ok when they get deserialized after an update to a version containing this code.